File: 60/Problem 3.c
#include <stdio.h>
int main(void) {
	int cases, i, n, m, arr[1001], q, r, sum;
	scanf("%d", &cases);
	while(cases--){
	    scanf("%d%d", &n, &m);
	    for(i = 0; i < n; i++){
	        scanf("%d", &arr[i]);
	    }
	    while(m--){
	        scanf("%d%d", &q, &r);
	        sum = 0;
	        q--;
	        r--;
	        for(;q <= r; q++){
	            sum += arr[q];
	        }
	        printf("%d\n", sum);
	    }
	}
	return 0;
}

File: 03/Problem 3.cs
using System;

namespace ConsoleApp2
{
    class Program
    {
        static void Main(string[] args)
        {
            int cases = int.Parse(Console.ReadLine());
            while(cases > 0)
            {
                string[] input = Console.ReadLine().Split(' ');
                ulong n = ulong.Parse(input[0]) - 1;
                ulong m = ulong.Parse(input[1]);
                ulong sum = (m * (m + 0x1) * (0x2 * m + 0x1) / 0x6) - (n * (n + 0x1) * (0x2 * n + 0x1) / 0x6);
                Console.WriteLine(sum);
                cases--;
            }
        }
    }
}
